### PR TITLE
Automap Improvements

### PR DESCRIPTION
**Pull request by SarahCosmosys**
This PR adds the ability to zoom with the mousewheel on the automap. It also adds the option to hide all stats from both the automap and overlay HUD and makes it so hu_hidesecrets only hides secrets.

### CODE REVIEW

File: source/am_map.cpp [hunk 1]
```diff
@@ -877,6 +877,36 @@ bool AM_Responder(const event_t *ev)
       if(ev->type != ev_keydown)
          return false;
 
+      // mousewheel zooming
+      if (ev->data1 == KEYD_MWHEELUP)
+      {
+         scale_mtof = scale_mtof * 1.3;
```
Comment (Altazimuth) [inline source/am_map.cpp:883]: Where's 1.3 come from?

Comment (SarahCosmosys) [reply source/am_map.cpp:883]: Sensitivity value for how much each scroll zooms in or out. I messed around until I found a decent value. I could add an option for it instead.

Comment (Altazimuth) [reply source/am_map.cpp:883]: Ah, figured as much, thanks. Honestly I think substituting it out for a constexpr is probably fine, both for the sake of clarity and also to make it easier if people want it as an option.

**Comment by SarahCosmosys** [comment]
Is there anything further needed before this can be merged?

**Comment by Altazimuth** [comment]
Nope!  This was something I'd routinely remember while in bed in the evening and think "Oh yeah I need to do that." Apologies for that, and thanks for the PR.

**Pull request merged**